protocol/src/main/protobuf/Transactor.proto

```diff
@@ -14,6 +14,24 @@ message UpdateRequest {
     bytes chainCellCbor = 1;
 }
 
+message JournalStreamRequest {
+    MultihashReference lastJournalBlock = 1;
+}
+
+message UpdateChainResult {
+    MultihashReference canonicalRef = 1;
+    MultihashReference chainHeadRef = 2;
+    MultihashReference chainPreviousHeadRef = 3;
```

Comment:
no, I think you're right.  using Ref is inconsistent with the other protos as well